fastapi_template/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/settings.py

```diff
@@ -62,6 +62,10 @@ class Settings(BaseSettings):
     sentry_sample_rate: float = 1.0
     {%- endif %}
 
+    {%- if cookiecutter.otlp_enabled == "True" %}
+    opentelemetry_endpoint: Optional[str] = None
```

Comment:
I wonder, if we set `otlp_enabled` to `True` wouldn't we want a default endpoint instead of None ?

Reply:
It works like this. 
If you don't provide any endpoint through environment variables, it won't enable the integration by default, but if you do it will enable tracing and start sending data to the specified endpoint.

Reply:
Also you don't always want to trace requests while running local copy of a project.